Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestJobsPanel.java

```diff
@@ -0,0 +1,210 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.experimental.autoingest;
+
+import java.awt.Dimension;
+import java.beans.PropertyVetoException;
+import javax.swing.ListSelectionModel;
+import javax.swing.event.ListSelectionListener;
+import org.netbeans.swing.outline.DefaultOutlineModel;
+import org.netbeans.swing.outline.Outline;
+import org.openide.explorer.ExplorerManager;
+import org.openide.nodes.Node;
+import org.openide.util.NbBundle.Messages;
+import org.sleuthkit.autopsy.datamodel.EmptyNode;
+import org.sleuthkit.autopsy.experimental.autoingest.AutoIngestJobsNode.AutoIngestJobStatus;
+import org.sleuthkit.autopsy.experimental.autoingest.AutoIngestJobsNode.JobNode;
+import org.sleuthkit.autopsy.experimental.autoingest.AutoIngestMonitor.JobsSnapshot;
+
+/**
+ * A panel which displays an outline view with all jobs for a specified status.
+ */
+final class AutoIngestJobsPanel extends javax.swing.JPanel implements ExplorerManager.Provider {
+
+    private static final long serialVersionUID = 1L;
+    private static final int INITIAL_CASENAME_WIDTH = 170;
+    private static final int INITIAL_DATASOURCE_WIDTH = 270;
+    private static final int INITIAL_PRIORITIZED_WIDTH = 20;
+    private static final int INITIAL_STATUS_WIDTH = 20;
+    private final org.openide.explorer.view.OutlineView outlineView;
+    private final Outline outline;
+    private ExplorerManager explorerManager;
+    private final AutoIngestJobStatus status;
+
+    /**
+     * Creates a new AutoIngestJobsPanel of the specified jobStatus
+     *
+     * @param jobStatus the status of the jbos to be displayed on this panel
+     */
+    AutoIngestJobsPanel(AutoIngestJobStatus jobStatus) {
+        initComponents();
+        status = jobStatus;
+        outlineView = new org.openide.explorer.view.OutlineView();
+        outline = outlineView.getOutline();
+        customize();
+    }
+
+    /**
+     * Set up the AutoIngestJobsPanel's so that its outlineView is displaying
+     * the correct columns for the specified AutoIngestJobStatus
+     */
+    @Messages({"AutoIngestJobsPanel.waitNode.text=Please wait..."})
+    void customize() {
+        ((DefaultOutlineModel) outline.getOutlineModel()).setNodesColumnLabel(Bundle.AutoIngestJobsNode_caseName_text());
+        int indexOfColumn;
+        switch (status) {
+            case PENDING_JOB:
+                outlineView.setPropertyColumns(Bundle.AutoIngestJobsNode_dataSource_text(), Bundle.AutoIngestJobsNode_dataSource_text(),
+                        Bundle.AutoIngestJobsNode_jobCreated_text(), Bundle.AutoIngestJobsNode_jobCreated_text(),
+                        Bundle.AutoIngestJobsNode_priority_text(), Bundle.AutoIngestJobsNode_priority_text());
+                indexOfColumn = getColumnIndexByName(Bundle.AutoIngestJobsNode_priority_text());
+                if (indexOfColumn != -1) {
```

Comment:
We should probably use a constant instead of just "-1". Something like "INVALID_COLUMN" or something.